GPU/Common/TextureCacheCommon.cpp

```diff
@@ -1353,7 +1355,10 @@ static inline void ConvertFormatToRGBA8888(GEPaletteFormat format, u32 *dst, con
 }
 
 template <typename DXTBlock, int n>
-static void DecodeDXTBlock(uint8_t *out, int outPitch, uint32_t texaddr, const uint8_t *texptr, int w, int h, int bufw, bool reverseColors, bool useBGRA) {
+static void DecodeDXTBlocks(uint8_t *out, int outPitch, uint32_t texaddr, const uint8_t *texptr,
+	int w, int h, int bufw, bool reverseColors, bool useBGRA,
+	u32 *alphaSum, u32 *fullAlphaMask) {
```

Comment:
nit: Maybe we should use a struct.  My general rule of thumb is if I have so many arguments that I'm motivated to put some on another line, then I have too many arguments.

-[Unknown]

Reply:
Yeah, it's probably time to clean this up.

Reply:
Removed the two parameters again, good enough for now heh

Reply:
Not gonna bother for now though.. I got rid of the new parameters from the calling function `DecodeTextureLevel` though.